extensions/blocks/contact-info/style.native.scss

```diff
@@ -0,0 +1,19 @@
+@import "~@wordpress/base-styles/colors";
+
+.placeholder {
+  color: $gray;
+}
+
+.placeholderDark {
+  color: $gray-50;
+}
+
+.blockEditorPlainTextColor {
+	color: $gray-900;
+	text-decoration-color: $blue-500;
+}
+
+.blockEditorPlainTextColorDark {
+	color: $white;
+	text-decoration-color: $blue-30;
+}
```

Comment:
One tip: You can define de default light color directly on `.blockEditorPlainText` (from `editor.native.scss`), and then define `.blockEditorPlainTextColorDark` as you have it here, just overwriting the colors for Dark theme.

`getStylesFromColorScheme` will be intelligent enough to just overwrite the colors and keep all other properties as they are.

This is only a suggestion since it works perfectly as it is now 👍

Reply:
Ahhh I didn't realize that -- that sounds great! Simplifying code always worth 👍

Reply:
For the above comment: Unfortunately, I'm still having issues with the font weight looking different between `RichText` and `Contact Info` in light mode on iOS, which is causing the placeholder color and text color to look different. On Android, the weight looks the same but the font sizes look different. Still investigating that issue as well as adding UI tests!

Reply:
> I'm still having issues with the font weight looking different between RichText and Contact Info in light mode on iOS

You are right. I completely missed that! 🙏 
I agree let's fix this too ;)

Reply:
Hello Jetpack devs! Is there an existing way in a jetpack component to share attribute lists between native and web code? Right now there are a lot of duplications between the two in `jetpack/extensions/blocks/contact-info/address/edit.js` and `jetpack/extensions/blocks/contact-info/address/edit.native.js`. Please see [lint issues 3-4](https://codeclimate.com/github/Automattic/jetpack/pull/17123)

Reply:
Would it be an option to create a new file just for the attribute list? Something like this:
https://github.com/Automattic/jetpack/blob/239483223dbe55f2f9959ea52ae04819391f1b3e/extensions/blocks/revue/attributes.js

Reply:
Hrmmm unfortunately in this case it's getting the attributes from the `props`. I could pull that out into a common file and then do `attributes.mem1 || ''`, etc but that seems unnecessarily complex and unsafe.

Reply:
Actually @jeherve if you could take a look at the 4 duplication errors on [lint](https://codeclimate.com/github/Automattic/jetpack/pull/17123) in general I'd be grateful :) Pulling out the duplication in `PhoneEdit`/`EmailEdit` into `CommonChildEdit` also seems to trigger another duplication lint error saying that using the child block is too similar lol.

Reply:
Oh, ok, I should have clicked on the link to find out more before I comment :\

I think you can ignore those warnings, I don't think it makes a lot of sense to refactor everything here.

In general, we view CodeClimate recommendations as non-blocking comments; they can be useful to point out potential areas of improvement, but at times they can be useless and too nit-picky.

Reply:
Thank you @jeherve for your insight!

I think we can do that, but then we still would need to expand the same attributes in two different files.
Something like this being repeated (from your example): https://github.com/Automattic/jetpack/blob/239483223dbe55f2f9959ea52ae04819391f1b3e/extensions/blocks/revue/edit.js#L42-L52

And probably this will be catch as code duplication too?

About the second lint error: @illusaen also created a `Common` component to avoid code duplication (https://github.com/Automattic/jetpack/pull/17123/commits/8315acfc0471e1cc38e381120c4b23ae33371556). The usage of this component looks very similar but they are two different instances with different setup. CodeClimate is catching this as code duplication too, which I find unexpected. Is there a known way of avoiding this?

Thank you for your help!

Reply:
> CodeClimate is catching this as code duplication too, which I find unexpected. Is there a known way of avoiding this?

I think you can just ignore those for now.

Reply:
Thanks for your help! 👍